src/components/player.js

```diff
@@ -72,6 +72,24 @@ class Player extends vjsPlayer {
     return audioTrack;
   }
 
+  /**
+   * Calculates an array of ranges based on the `seekable()` data.
+   *
+   * @returns {Array} An array of objects representing start and end points of seekable ranges.
```

Comment:
We could further describe the type returned by the Array:

```suggestion
   * @returns {Array<{start: number, end: number}>} An array of objects representing start and end points of seekable ranges.
```